### ISSUE TITLE
Still possible to hit "enter" to submit / advance form when conditional logic hides it

### ISSUE THREAD

**Issue by brycekirk** [verbb/formie#1136]
### Describe the bug

I have a submit button with some conditional logic on it, basically, a field must match certain values. This works as expected, ie, the submit button shows / hides according to my set logic, but you're able to bypass all of this by simply hitting "enter". This is possible whether any conditional logic is met or not

### Steps to reproduce

1. Add a text field to a form
2. Add conditional logic to the submit button so that the text field needs to match a certain value
3. Check that the logic is working to show / hide the submit button
4. Hit "enter" while the button is hidden
5. The form still submits

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

Craft Solo 4.2.8

### Plugin version

2.0.12

### Multi-site?

No

**Comment by brycekirk** [comment]
Seems like perhaps [this is intentional](https://verbb.io/craft-plugins/formie/docs/feature-tour/conditions#buttons)? 

Got around it by listening for changes to my text input and disabling the submit button as long as `data-conditionally-hidden 
 == true`.

**Comment by engram-design** [comment]
Hmmm, that's a good point in that the button isn't disabled when conditionally hidden. It's arguable that it should be, so I'll give this some further thought!

**Comment by sunnylawn** [comment]
Yeah, i also need the facility to hide the submit button until certain conditions are met, or disable it visually.

**Comment by engram-design** [comment]
We'll spin this out into a new issue, where we can add conditional logic to buttons to show and hide them based on the regular field conditions, so you can control their visibility.

**Comment by sunnylawn** [comment]
Thanks, just had it direct from client who's testing some multipage form, its almost needing next/previous for multipage form setups and then a submit that allows for dedicated conditionals.

**Comment by engram-design** [comment]
> We'll spin this out into a new issue, where we can add conditional logic to buttons to show and hide them based on the regular field conditions, so you can control their visibility.

After writing that comment, I completely forgot that we currently **do** have page button conditionals available as a feature! But maybe you're referring to the Submit button specifically, not all the buttons (Save and Back).

But yes, I'll take a look at adding the `disabled` attribute.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.7](https://github.com/verbb/formie/releases/tag/4.0.0-beta.7).
